tests/integration/test_dataset_routes.py

```diff
@@ -0,0 +1,47 @@
+import pytest
+from tests.fixtures import initialize_test_db, api_client
+
+
+@pytest.mark.skip()
```

Comment:
If I understand pytest.mark.skip() correctly, this skips the test alltogether. Is this the intended behaviour?

Reply:
Thanks for catching that! 

Fixed the tests in the latest commits.